### ISSUE TITLE
Same issue as #1364 and #3742 on Jellyfin 10.6.0 thru 1.6.4  also on 1.7.0_rc1

### ISSUE THREAD

**Issue by shorewall** [jellyfin/jellyfin#4848]
Jellyfin browser tab stuck if Jellyfin fails to reproduce an hls flow issuing exit code 1.
This link works perfectly using a simple plain ffmpeg cmd:
ffmpeg -i 'http://mediapolisvod.rai.it/relinker/relinkerServlet.htm?cont=SaDv0BcoDFceeqqEEqual' -f mpegts - | mpv -

Mind this is a public RAI (Italian National TV) link and you need an italian ip to make it works.
Jellyfin fails to reproduce (See attached log) giving exit code 1.
The browser get the Jellyfin tab locked. Cannot go back to a working menu.
(See attached screenshot).
I must close the browser tab.

**System (please complete the following information):**
 - OS: Linux Mageia 6.0  6.1
 - Clients: [Browser]
 - Browser: Palemoon 28.11.0 x64,  Firefox 64.2.0 X64
 - Jellyfin Version: 1.7.0_rc1
 - Playback: [Direct Play, Remux, Direct Stream, Transcode] 
 - Installed Plugins: [e.g. none, Fanart, Anime, etc.]
 - Base URL: [e.g. none, yes: /example]
 - Networking: [e.g. Host, Bridge/NAT]
 - Storage: [e.g. local, NFS, cloud]

**To Reproduce**

1. config Jellyfin to reproduce this hls flow: http://mediapolisvod.rai.it/relinker/relinkerServlet.htm?cont=SaDv0BcoDFceeqqEEqual  as Tv channel.
2. Start stream
3. The browser tab remains black and hung.

**Expected behavior**
I expect a goback arrow to make another choice.

**Logs**

[14:54:35] [INF] [9] Jellyfin.Api.Helpers.MediaInfoHelper: Profile: Unknown Profile, No video direct play profiles found for http://mediapolisvod.rai.it/relinker/relinkerServlet.htm?cont=SaDv0BcoDFceeqqEEqual with codec h264
[14:54:36] [INF] [9] Jellyfin.Api.Helpers.TranscodingJobHelper: /opt/FFMpeg/ffmpeg -analyzeduration 3000000 -user_agent "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.85 Safari/537.36" -fflags +igndts -f hls -i "http://mediapolisvod.rai.it/relinker/relinkerServlet.htm?cont=SaDv0BcoDFceeqqEEqual" -map_metadata -1 -map_chapters -1 -threads 0 -sn -codec:v:0 libx264 -pix_fmt yuv420p -preset superfast -crf 23 -maxrate 139808000 -bufsize 279616000 -profile:v:0 high -level 41 -x264opts:0 subme=0:me_range=4:rc_lookahead=10:me=dia:no_chroma_me:8x8dct=0:partitions=none  -force_key_frames:0 "expr:gte(t,n_forced*3)" -start_at_zero -flags -global_header -vsync -1 -acodec copy -strict -2 -copyts -avoid_negative_ts disabled -max_muxing_queue_size 2048 -f hls -max_delay 5000000 -hls_time 3 -hls_segment_type mpegts -start_number 0 -hls_base_url "hls/2afcaa3c36857d4ddc62ac6c6871bd3d/" -hls_playlist_type event -hls_segment_filename "/root/.local/share/jellyfin/transcodes/2afcaa3c36857d4ddc62ac6c6871bd3d%d.ts" -y "/root/.local/share/jellyfin/transcodes/2afcaa3c36857d4ddc62ac6c6871bd3d.m3u8"
[14:54:36] [ERR] [9] Jellyfin.Api.Helpers.TranscodingJobHelper: FFmpeg exited with code 1

**Screenshots**
![Jellyfin_browser_lock](https://user-images.githubusercontent.com/15981554/102897828-ebe6ef80-4468-11eb-9ace-e449d4a6077f.png)

![Jellyfin_palemoon](https://user-images.githubusercontent.com/15981554/102898958-901d6600-446a-11eb-9fa8-3ed49d70d4b8.png)

**Additional context**
[Jellyfin.zip](https://github.com/jellyfin/jellyfin/files/5730303/Jellyfin.zip)

**Comment by crobibero** [comment]
Can you also please attach the ffmpeg log?

**Comment by shorewall** [comment]
[FFmpeg.log](https://github.com/jellyfin/jellyfin/files/5730406/FFmpeg.log)
[FFmpeg.zip](https://github.com/jellyfin/jellyfin/files/5730407/FFmpeg.zip)

**Comment by shorewall** [comment]
Here the ffmpeg log which works starting by hand

**Comment by crobibero** [comment]
Thanks for that, but can you also please attach the jellyfin ffmpeg log?

**Comment by shorewall** [comment]
Pls could you tell me how to get or where to find the Jellyfin ffmpeg log?

**Comment by crobibero** [comment]
https://jellyfin.org/docs/general/administration/configuration.html#log-directory

You can also view the log directory from the dashboard, under Paths

**Comment by shorewall** [comment]
/videos/7fbda47a-b322-6062-d625-dac1959c696e/live.m3u8

{"Protocol":1,"Id":"9d3e548282bdeeaf329b6e861975c5a9","Path":"http://mediapolisvod.rai.it/relinker/relinkerServlet.htm?cont=SaDv0BcoDFceeqqEEqual","EncoderPath":null,"EncoderProtocol":null,"Type":0,"Container":"hls","Size":290,"Name":null,"IsRemote":true,"ETag":null,"RunTimeTicks":null,"ReadAtNativeFramerate":false,"IgnoreDts":true,"IgnoreIndex":false,"GenPtsInput":false,"SupportsTranscoding":true,"SupportsDirectStream":true,"SupportsDirectPlay":true,"IsInfiniteStream":true,"RequiresOpening":true,"OpenToken":null,"RequiresClosing":true,"LiveStreamId":"a17c75760a04e99b68cf766e11316e1c_09efa0d56b934a82adec00a87b837fb0_9d3e548282bdeeaf329b6e861975c5a9","BufferMs":null,"RequiresLooping":false,"SupportsProbing":true,"VideoType":null,"IsoType":null,"Video3DFormat":null,"MediaStreams":[{"Codec":"h264","CodecTag":null,"Language":null,"ColorRange":null,"ColorSpace":null,"ColorTransfer":null,"ColorPrimaries":null,"Comment":null,"TimeBase":"1/90000","CodecTimeBase":"1/50","Title":null,"VideoRange":"SDR","localizedUndefined":null,"localizedDefault":null,"localizedForced":null,"DisplayTitle":"SD H264 SDR","NalLengthSize":"0","IsInterlaced":false,"IsAVC":null,"ChannelLayout":null,"BitRate":null,"BitDepth":8,"RefFrames":1,"PacketLength":null,"Channels":null,"SampleRate":null,"IsDefault":false,"IsForced":false,"Height":288,"Width":512,"AverageFrameRate":25,"RealFrameRate":25,"Profile":"Main","Type":1,"AspectRatio":"16:9","Index":-1,"Score":null,"IsExternal":false,"DeliveryMethod":null,"DeliveryUrl":null,"IsExternalUrl":null,"IsTextSubtitleStream":false,"SupportsExternalStream":false,"Path":null,"PixelFormat":"yuv420p","Level":31,"IsAnamorphic":false},{"Codec":"aac","CodecTag":null,"Language":null,"ColorRange":null,"ColorSpace":null,"ColorTransfer":null,"ColorPrimaries":null,"Comment":null,"TimeBase":"1/90000","CodecTimeBase":"1/44100","Title":null,"VideoRange":null,"localizedUndefined":null,"localizedDefault":null,"localizedForced":null,"DisplayTitle":"AAC - Stereo","NalLengthSize":null,"IsInterlaced":false,"IsAVC":null,"ChannelLayout":"stereo","BitRate":192000,"BitDepth":null,"RefFrames":null,"PacketLength":null,"Channels":2,"SampleRate":44100,"IsDefault":false,"IsForced":false,"Height":null,"Width":null,"AverageFrameRate":null,"RealFrameRate":null,"Profile":"LC","Type":0,"AspectRatio":null,"Index":-1,"Score":null,"IsExternal":false,"DeliveryMethod":null,"DeliveryUrl":null,"IsExternalUrl":null,"IsTextSubtitleStream":false,"SupportsExternalStream":false,"Path":null,"PixelFormat":null,"Level":0,"IsAnamorphic":null}],"MediaAttachments":[],"Formats":[],"Bitrate":192000,"Timestamp":null,"RequiredHttpHeaders":{"User-Agent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.85 Safari/537.36"},"TranscodingUrl":null,"TranscodingSubProtocol":null,"TranscodingContainer":null,"AnalyzeDurationMs":3000,"DefaultAudioStreamIndex":null,"DefaultSubtitleStreamIndex":null}

/opt/FFMpeg/ffmpeg -analyzeduration 3000000 -user_agent "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.85 Safari/537.36" -fflags +igndts -f hls -i "http://mediapolisvod.rai.it/relinker/relinkerServlet.htm?cont=SaDv0BcoDFceeqqEEqual" -map_metadata -1 -map_chapters -1 -threads 0 -sn -codec:v:0 libx264 -pix_fmt yuv420p -preset superfast -crf 23 -maxrate 139808000 -bufsize 279616000 -profile:v:0 high -level 41 -x264opts:0 subme=0:me_range=4:rc_lookahead=10:me=dia:no_chroma_me:8x8dct=0:partitions=none  -force_key_frames:0 "expr:gte(t,n_forced*3)" -start_at_zero -flags -global_header -vsync -1 -acodec copy -strict -2 -copyts -avoid_negative_ts disabled -max_muxing_queue_size 2048 -f hls -max_delay 5000000 -hls_time 3 -hls_segment_type mpegts -start_number 0 -hls_base_url "hls/c350f75462ddda3d6ae70bcbcfcc2aea/" -hls_playlist_type event -hls_segment_filename "/root/.local/share/jellyfin/transcodes/c350f75462ddda3d6ae70bcbcfcc2aea%d.ts" -y "/root/.local/share/jellyfin/transcodes/c350f75462ddda3d6ae70bcbcfcc2aea.m3u8"


ffmpeg version 4.3.1 Copyright (c) 2000-2020 the FFmpeg developers
  built with gcc 5.5.0 (Mageia 5.5.0-1.mga6)
  configuration: --enable-gnutls
  libavutil      56. 51.100 / 56. 51.100
  libavcodec     58. 91.100 / 58. 91.100
  libavformat    58. 45.100 / 58. 45.100
  libavdevice    58. 10.100 / 58. 10.100
  libavfilter     7. 85.100 /  7. 85.100
  libswscale      5.  7.100 /  5.  7.100
  libswresample   3.  7.100 /  3.  7.100
Unrecognized option 'preset'.
Error splitting the argument list: Option not found

**Comment by shorewall** [comment]
[JellyfinFFmpeg.zip](https://github.com/jellyfin/jellyfin/files/5730496/JellyfinFFmpeg.zip)
Here the Jellyfin own ffmpeg log

**Comment by shorewall** [comment]
Update: the ffmpeg bug (preset option non found) is solved by recompiling from source in the followinf way:
ffmpeg version N-100402-g0320dab Copyright (c) 2000-2020 the FFmpeg developers
  built with gcc 5.5.0 (Mageia 5.5.0-1.mga6)
  configuration: --enable-gnutls --enable-libxml2 --enable-libx264 --enable-gpl
  libavutil      56. 62.100 / 56. 62.100
  libavcodec     58.115.102 / 58.115.102
  libavformat    58. 65.100 / 58. 65.100
  libavdevice    58. 11.103 / 58. 11.103
  libavfilter     7. 93.100 /  7. 93.100
  libswscale      5.  8.100 /  5.  8.100

Enabling the libx264 included the missing preset.
But doesn't means that the bug I opened this issue for has been solved.
The same browser tab lock can happens facing a wrong FFmpeg exit error 1 on any other link.
And is very problematic close the browser tab and reacces Jellyfin to gain again Jellyfin's control.
So I would keep this bug open until fully fixed.
